packages/core/admin/admin/src/content-manager/hooks/useLazyComponents/tests/useLazyComponents.test.js

```diff
@@ -72,4 +91,25 @@ describe('useLazyComponents', () => {
     expect(result.current.isLazyLoading).toEqual(false);
     expect(result.current.lazyComponentStore).toEqual({});
   });
+
+  test('assuming the store has been initialised before hand, other hooks called should be able to modify the global cache and access it', async () => {
```

Comment:
This test looks really similar to the one on line 70. Does it need to be its own test case? Couldn't we just extend the existing test to make sure the cache can be accessed and modified?